React/src/screens/Dashboard/screens/Home/styles.module.scss

```diff
@@ -1,39 +1,78 @@
-.app {
-  text-align: center;
-}
-
-.app-logo {
-  animation: app-logo-spin infinite 20s linear;
-  height: 30vmin;
-}
-
-.app-header {
-  background-color: #282C34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-}
-
-.text {
-  font-size: calc(10px + 2vmin);
-  margin-bottom: calc(30px + 2vmin);
-  color: #FFF;
-}
-
-.app-link {
-  font-size: calc(10px + 2vmin);
-  color: #61DAFB;
-  text-decoration: underline;
-}
-
-@keyframes app-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
+@import '../../../../scss/variables/colors';
+@import '../../../../scss/mixins.scss';
+
+.pageContainer {
+    @include flexBox();
+
+    .loginContainer {
+      align-items: center;
+      background-color: $seashell;
+      border-top: 10px solid $cerulean;
+      display: flex;
+      flex-direction: column;
+      height: 700px;
+      margin-top: 80px;
+      width: 400px;
+
+      img {
```

Comment:
Use clases instead.
```suggestion
      .woloxLogo {
```

Reply:
The same, I tried to put a class to my component "Wolox Logo" but the styles do not apply to the image